bridge-stp.in

```diff
@@ -28,31 +28,51 @@ net_dir=/sys/class/net
 # the kernel's STP implementation.  If empty, MSTP will be used for all bridges.
 MSTP_BRIDGES=''
 
+# Determine whether mstpd should manage STP for the specified bridge.
+# Returns 0 if mstpd should manage STP for the specified bridge, or 1 if mstpd
+# should not manage STP for the specified bridge.
+is_mstp_bridge()
```

Comment:
@PaulSD 
ack for this change too
question: are you going to re-spin that change regarding the "Coudln't connect to server" ? 

or are you dropping that change ?
fine either way from me